.github/workflows/gradle.yml

```diff
@@ -0,0 +1,13 @@
+name: Gradle Tests
+on: [push]
+jobs:
+  gradle_test:
+    name: Run tests with Gradle
+    runs-on: ubuntu-latest
+    container: gradle:6.6-jre14-hotspot
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Run Gradle tests
+      run: gradle --console plain
```

Comment:
So on AppVeyor, we currently do:

```
gradlew.bat --no-daemon assemble
gradlew.bat --no-daemon check
```

I'm not sure where you found `gradle --console plain`, and I'm no gradle wizard myself, so I'm not 100% sure what that does... but I did try it locally and it seemed to run the tests. :shrug: Where did this come from?

Reply:
`--console plain` is to remove any animations that are aimed at the terminal but not at a logfile.

Apart from that, I disliked the use of gradlew/gradlew.bat in a different project (putting generated code into version control), which is why I searched for a way to run this inside a Docker container. Actually, you don't need those two wrapper scripts then. Also, locally, I keep a running Docker container just for the daemon to speed up subsequent tests.